central/group/datastore/datastore.go

```diff
@@ -18,9 +18,9 @@ type DataStore interface {
 
 	Add(ctx context.Context, group *storage.Group) error
 	Update(ctx context.Context, group *storage.Group) error
-	Mutate(ctx context.Context, remove, update, add []*storage.Group) error
-	Remove(ctx context.Context, props *storage.GroupProperties) error
-	RemoveAllWithAuthProviderID(ctx context.Context, authProviderID string) error
+	Mutate(ctx context.Context, force bool, remove, update, add []*storage.Group) error
```

Comment:
nit but I would find it more logical if `force` were the last parameter. I think that's also the ordering I have seen in other methods, such as the helper methods introduced in this PR